package.json

```diff
@@ -75,7 +75,8 @@
     "react-native": ">=0.59"
   },
   "dependencies": {
-    "invariant": "^2.2.4"
+    "invariant": "^2.2.4",
+    "react-native-appstate-hook": "^1.0.4"
```

Comment:
this would be a development dep only, because it is not used by the library itself but just the example app